src/test/ui/print_type_sizes/nullable.stdout

```diff
@@ -0,0 +1,24 @@
+print-type-size type: `IndirectNonZero<u32>` overall bytes: 40 align: 8
+print-type-size    field .pre bytes: 8
+print-type-size    field .nested bytes: 24
+print-type-size    field .post bytes: 8
+print-type-size type: `MyOption<IndirectNonZero<u32>>` overall bytes: 40 align: 8
+print-type-size    variant Some exact bytes: 40
+print-type-size        field .0 bytes: 40
```

Comment:
(I'll go ahead and try putting in all of the suggestions and see how the output fares. I think my biggest issue might be with adding backticks to the variants and fields, but I'm willing to give it a shot.)